openid-connect-server/src/test/java/org/mitre/oauth2/service/impl/TestDefaultIntrospectionResultAssembler.java

```diff
@@ -149,8 +149,8 @@ public void shouldAssembleExpectedResultForAccessTokenWithoutUserInfo() throws P
 		assertThat(result, is(equalTo(expected)));
 	}
 
-	@Test
-	public void shouldAssembleExpectedResultForAccessTokenWithoutExpiry() {
+	@Test (expected = ParseException.class)
```

Comment:
This test should not throw an exception, so why are you saying that an exception was expected? Leave this test as-is.